docs/local_development.md

```diff
@@ -105,25 +105,15 @@ Use `invoke npm update`.
 
 Some development work requires testing changes against "whatever the current production database looks like", which requires having postgresql installed locally (`brew install postgresql` on mac; download and run the official installer for windows; if you use linux/unix, you know how to install things for your favourite flavour, so just do that for postgresql). We backport prod data to staging every week, scrubbing PII, so we'll be creating a copy of that for local testing, too.
```

Comment:
We probably want to add a bit to make it explicit this relies on the Heroku cli, with a link to its download page.

Reply:
yeah, I'll add in something to that effect.